core/src/test/java/io/kestra/core/storages/InternalStorageTest.java

```diff
@@ -16,44 +16,44 @@
 class InternalStorageTest {
 
 
-    @Test
```

Comment:
Wy is it commented out?
If it cannot be run, remove them

Reply:
I forgot to remove them